gradio/__init__.py

```diff
@@ -67,5 +67,5 @@
     Webcam,
 )
 
-current_pkg_version = pkg_resources.require("gradio")[0].version
+current_pkg_version = version("gradio")
```

Comment:
Use pkg_resources is discouraged by setuptools

![image](https://user-images.githubusercontent.com/41651716/176771809-fc0213c2-afc9-4615-9b6b-433403e8f442.png)